R/use_standalone.R

```diff
@@ -182,7 +182,7 @@ standalone_dependencies <- function(lines, path, error_call = caller_env()) {
   imports <- as_chr_field(yaml$imports)
   imports <- as_version_info(imports, error_call = error_call)
 
-  if (any(na.omit(imports$cmp) != ">=")) {
+  if (any(stats::na.omit(imports$cmp) != ">=")) {
```

Comment:
This has nothing to do with this PR, but I've been getting a NOTE every time I run check() locally :)

Reply:
Yeah I noticed this reading a GHA log today 👍